helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java

```diff
@@ -136,46 +113,6 @@ private static synchronized HeliosDeployment getOrCreateHeliosSoloDeployment() {
     return soloDeployment;
   }
 
-  /**
-   * Perform setup. This is normally called by JUnit when TemporaryJobs is used with @Rule.
-   * If @Rule cannot be used, call this method before calling {@link #job()}.
-   *
-   * Note: When not being used as a @Rule, jobs will not be monitored during test runs.
-   */
-  public void before() {
-    deployer.readyToDeploy();
```

Comment:
The coupling between `DefaultDeployer` and `TemporaryJobsResource` is pretty weird.

Note that `TemporaryJobs` is currently not usable without `TemporaryJobsResource` (== I think this commit needs some more work). If you try to use `TemporaryJobs` with `TemporaryJobsResource` then `Deployer.readyToDeploy()` will not be called, which means that `TemporaryJobs.deploy()` will throw an exception.